server/statistics/kind.go

```diff
@@ -97,6 +97,27 @@ func (k StoreStatKind) String() string {
 	return "unknown StoreStatKind"
 }
 
+// SourceKind represents the statistics item source.
+type SourceKind int
+
+const (
+	Origin  SourceKind = iota // there is a corresponding peer in this store
+	Inherit                   // there is no a corresponding peer in this store and there is a peer just deleted
+	Adopt                     // there is no a corresponding peer in this store and there is no a peer just deleted, we need to copy from other store
+)
+
+func (k SourceKind) String() string {
+	switch k {
+	case Origin:
+		return "origin"
+	case Inherit:
```

Comment:
What is `Inherit` used for? It looks we only need `Adopt` right now.

Reply:
yep, only distinct different source